index.d.ts

```diff
@@ -0,0 +1,5 @@
+export default class IncludeFragmentElement extends HTMLElement {
+  get data(): Promise<string>;
```

Comment:
```
Error	TypeScript	An accessor cannot be declared in an ambient context.
```

Reply:
Yup! I was reading about this a lot last night and it took a while to figure out. I wanted to change this to a `readonly` but that would give me weird errors as well. Turns out that I had a really old VSCode installed 😱